docs/docs/guides/docker-help.md

```diff
@@ -4,11 +4,27 @@ sidebar_position: 1
 
 # Docker Help
 
-## Logs
+## Containers
 
-```bash title="Log Examples"
+```bash
 docker ps                         # see a list of running containers
 docker ps -a                      # see a list of running and stopped containers
+```
+
+## Attach to a Container
+
+```bash
+docker exec -it <id or name> <command>          # attach to a container with a command
+docker exec -it immich_server bash
```

Comment:
Have you checked if `bash` is installed in the container, I am pretty sure `sh` is the available shell

Reply:
Oh, looks like you're right. Changed to `sh`.